ir/src/symbol_table.rs

```diff
@@ -32,6 +35,7 @@ impl Display for IdentifierType {
                 write!(f, "TraceColumns in segment {}", columns.trace_segment())
             }
             Self::IntegrityVariable(_) => write!(f, "IntegrityVariable"),
+            Self::RandomValuesBinding(_, _) => write!(f, "RandomValue"),
```

Comment:
minor nit: Should this be RandomValuesBinding?

Reply:
My bad, I overlooked it. Thank you!